test/generate_property_json.js

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env node
+
+const faker = require('faker')
+
+const nbInitOps = 16
+const nbRunOps = 64
+const specialChars = [':', '_', 'é', ' ', '%', '\\', '&', '@', 'É', 'Ç']
+
+let knownPaths = []
+let deletedPaths = []
+
+function similarPath () {
+  let p = faker.random.arrayElement(knownPaths)
+  let q = p.replace('é', '\u00E9')
+  if (p !== q) {
+    return q
+  }
+  q = p.toLowerCase()
+  if (p !== q) {
+    return q
+  }
+  q = p.toUpperCase()
+  if (p !== q) {
+    return q
+  }
+  return p + faker.random.arrayElement(specialChars)
+}
+
+function newPath () {
+  if (knownPaths.length > 0 && faker.random.number(4) === 0) {
+    return similarPath()
+  }
+  let p = faker.system.fileName()
+  if (faker.random.number(4) === 0) {
+    p = faker.random.arrayElement(specialChars) + p
+  }
+  if (knownPaths.length > 0 && faker.random.number(3) > 0) {
+    p = faker.random.arrayElement(knownPaths) + '/' + p
+  }
+  knownPaths.push(p)
+  return p
+}
+
+function knownPath () {
+  if (knownPaths.length === 0) {
+    return newPath()
+  }
+  return faker.random.arrayElement(knownPaths)
+}
+
+function deletedPath () {
+  if (deletedPaths.length === 0) {
+    return newPath()
+  }
+  return faker.random.arrayElement(deletedPaths)
+}
+
+function fileSize () {
+  return faker.random.number(32) + 2 ** (1 + faker.random.number(20))
+}
+
+function createNewDir () {
+  return { op: 'mkdir', path: newPath() }
```

Comment:
I explicitly choose to not call it `actions` to avoid the confusion with the `actions` in scenarios as they are not the same. Maybe, on day, we will find a way to unify those two, and using the same name will be logical then. But, for the moment, I prefer to stay on something else that `actions`.